configs/workload/bert.yaml

```diff
@@ -28,4 +28,4 @@ reader:
 
 checkpoint:
  steps_between_checkpoints: 1250
- model_size: 4034713312
```

Comment:
Revert this. I think it was done by mistake.

Reply:
The field is not removed.  The last line is not currently newline terminated.  https://github.com/argonne-lcf/dlio_benchmark/pull/20/files#diff-a71b531b049497d6eca434150d4abf41b058d3956ac2d3c715abd0e291d41ab9R31